frontend/src/home/EditPlanPopper.tsx

```diff
@@ -309,6 +310,24 @@ export class EditPlanPopperComponent extends React.Component<
     );
   }
 
+  renderResetToApprovedButton() {
```

Comment:
idk if we really need this tbh, gonna leave it in for now and might remove it in set primary plan PR